python/orion_finance_sdk_py/costs/venues/uniswap_v3/simulator.py

```
@@ -0,0 +1,250 @@
+    anchor = (state.meta.stable_token or "").lower()
+    t0 = state.meta.token0.lower()
+    t1 = state.meta.token1.lower()
+    if anchor == t0:
+        return 1.0, (1.0 / ratio if ratio else 0.0)
+    if anchor == t1:
+        return (ratio if ratio else 0.0), 1.0
+    raise ValueError(f"Pool {state.meta.address} is not a USDC pair")
+
+
+def _next_initialized_tick(
+    initialized_ticks: list[int], tick: int, zero_for_one: bool
```

Ruff (0.16.1)

[warning] Boolean-typed positional argument in function definition

(FBT001)